lib/shared/widgets/layer/layer_widget.dart

```diff
@@ -298,6 +298,8 @@ class _LayerWidgetState extends State<LayerWidget>
       callbacks: callbacks,
       selected: widget.selected,
       onEditLayer: widget.onEditTap,
+      forceIgnoreGestures:
```

Comment:
I'm not sure if that's going to be a problem. But if we use `IgnorePointer` every callback below, it might stop working too. I'm about 90% sure there won't be any issues with custom user logic, but i didn't test it. 
You mentioned before the only problem is with the double-tap interaction, right? What do you think about this approach: 
```suggestion
forceIgnoreGestures:
    !(interaction.enableSelection || interaction.enableEdit) &&
        configs.mainEditor.enableZoom &&
        configs.mainEditor.enableDoubleTapZoom,
```

If you're confident that no necessary callbacks will be blocked, you can ignore it, I'll still go ahead and merge it. I'll leave the decision up to you whether we should extend it or not.

Reply:
I didn't include those because the my idea was if layer interactions are disable allow everything else in the main editor like zoom on any gesture.